### ISSUE TITLE
Cactus update form does not create entity from backlog after force processing

### ISSUE THREAD

**Issue by WKobus** [getodk/central#811]
#### Problem description
Cactus update form does not create entity from backlog. The problem may be caused by the entity properties that are inside "Cacti" entity list, as when I tried same forms in different project entity was created successfully by update form from backlog.
#### URL of the page
https://staging.getodk.cloud/#/projects/103/entity-lists/cacti/entities
#### Steps to reproduce the problem
Preconditions:
Auto send set to off in Collect

1. Go to https://staging.getodk.cloud/#/projects/103/app-users and scan user "cactus"
2. Go to collect > start new form
3. Open Cactus registration form fill it and finalize
4. Open Cactus monitoring form select previously finalized entity, fill form and finalize
5. Go to ready to send, select and send previously finalized Cactus monitoring form
6. Force process entities from backlog on Central
7. Go to "Cacti" entity data tab

#### Expected behavior
Entity should be created from update form in backlog
#### Central version shown in version.txt
51a3e10fdf188473ce38b24313962c674188c9ff (v2024.2.1-4-g51a3e10)
+bb2b2325c5fde4e1325e0d0fcec9c3db6a58cfd7 client (v2024.2.1-35-gbb2b2325)
+57c4da581a21928eb21fef8097f72870af6e4d4d server (v2024.2.0-83-g57c4da58)
#### Browser version
Firefox 132.0 (64-bit)

**Comment by matthew-white** [comment]
@ktuite, this might be a good one for you to look into if you have time.

**Comment by ktuite** [comment]
I think this is because the dataset/entity list requires approval before submissions are created... and that blocks this update submission from being force-applied as a create.

Doesn't look like much but I tried with this submission:
(monitoring) https://staging.getodk.cloud/#/projects/103/forms/cactus_monitoring/submissions/uuid%3Aad789a12-a805-4039-b8ff-c76c0c8efeb2

(registration) https://staging.getodk.cloud/#/projects/103/forms/cactus_reg/submissions/uuid%3Ac7fdedfa-e5fe-473e-a987-893e9a06cf9b

I saw the monitoring submission in the backlog and when i ran the force-processing command, i saw it "process" 1 thing, though it ultimately decided to skip it.

While possibly confusing, I think this is the "expected" behavior.
